parts/linux/cloud-init/artifacts/sysctl-d-60-CIS.conf

```diff
@@ -1,24 +1,49 @@
-# 3.1.2 Ensure packet redirect sending is disabled
+# Ubuntu CIS Benchmark: 3.1.2 Ensure packet redirect sending is disabled
 net.ipv4.conf.all.send_redirects = 0
 net.ipv4.conf.default.send_redirects = 0
-# 3.2.1 Ensure source routed packets are not accepted 
+
+# Ubuntu CIS Benchmark: 3.2.1 Ensure source routed packets are not accepted
+# Mariner AKS CIS Benchmark: 3.3.1 Ensure source routed packets are not accepted
 net.ipv4.conf.all.accept_source_route = 0
 net.ipv4.conf.default.accept_source_route = 0
-# 3.2.2 Ensure ICMP redirects are not accepted
+net.ipv6.conf.all.accept_source_route = 0
+net.ipv6.conf.default.accept_source_route = 0
+
+# Ubuntu CIS Benchmark: 3.2.2 Ensure ICMP redirects are not accepted
+# Mariner AKS CIS Benchmark: 3.3.2 Ensure ICMP redirects are not accepted
 net.ipv4.conf.all.accept_redirects = 0
 net.ipv4.conf.default.accept_redirects = 0
-# 3.2.3 Ensure secure ICMP redirects are not accepted
+net.ipv6.conf.all.accept_redirects = 0
+net.ipv6.conf.default.accept_redirects = 0
+
+# Ubuntu CIS Benchmark: 3.2.3 Ensure secure ICMP redirects are not accepted
+# Mariner AKS CIS Benchmark: 3.3.3 Ensure secure ICMP redirects are not accepted
 net.ipv4.conf.all.secure_redirects = 0
 net.ipv4.conf.default.secure_redirects = 0
-# 3.2.4 Ensure suspicious packets are logged
+
+# Ubuntu CIS Benchmark: 3.2.4 Ensure suspicious packets are logged
+# Mariner AKS CIS Benchmark: 3.3.4 Ensure suspicious packets are logged
 net.ipv4.conf.all.log_martians = 1
 net.ipv4.conf.default.log_martians = 1
-# 3.3.1 Ensure IPv6 router advertisements are not accepted
+
+# Ubuntu CIS Benchmark: 3.3.1 Ensure IPv6 router advertisements are not accepted
+# Mariner AKS CIS Benchmark: 3.3.9 Ensure IPv6 router advertisements are not accepted
 net.ipv6.conf.all.accept_ra = 0
 net.ipv6.conf.default.accept_ra = 0
-# 3.3.2 Ensure IPv6 redirects are not accepted
-net.ipv6.conf.all.accept_redirects = 0
-net.ipv6.conf.default.accept_redirects = 0
+
+# Mariner AKS CIS Benchmark: 3.3.5 Ensure broadcast ICMP requests are ignored
```

Comment:
Be cautious with the recommendation numbers for the Mariner benchmark. If there are any that were removed in a section, they'll be renumbered at publication time.